npm_and_yarn/lib/dependabot/npm_and_yarn/file_updater/npm_lockfile_updater.rb

```diff
@@ -17,35 +17,22 @@ class NpmLockfileUpdater
         require_relative "npmrc_builder"
         require_relative "package_json_updater"
 
-        def initialize(dependencies:, dependency_files:, credentials:)
```

Comment:
Most of this is a re-write because we no longer need pass around `lockfile`

Reply:
That's a great improvement 👍